eng/Dependencies.props

```diff
@@ -197,6 +197,7 @@ and are generated based on the last package release.
     <LatestPackageReference Include="MessagePack" />
     <LatestPackageReference Include="MessagePackAnalyzer" />
     <LatestPackageReference Include="Microsoft.Data.SqlClient" />
+    <LatestPackageReference Include="Microsoft.Windows.CsWin32" />
```

Comment:
@Tratcher the issue is that we're not supposed to be using external dependencies in SourceBuild (see line 185), so if you're using this dep in HttpSys then it'd have to be excluded from SourceBuild (CC @MichaelSimons)

Reply:
HttpSys is windows only so source build shouldn't be relevant?

Reply:
In that case add `<ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>` to the project file

Reply:
will source build be irrelevant for Windows forever ? (idk)

Reply:
eh, disregard, it's not on the table in that sense (eg., Windows C++ toolset is closed source). brainstorm, was thinking of VMR